modules/util/source-formatter/src/test/java/com/liferay/source/formatter/processor/UpgradeSourceProcessorTest.java

```diff
@@ -127,6 +127,11 @@ public void testUpgradeJavaGetLayoutDisplayPageProviderCheck()
 		test("upgrade/UpgradeJavaGetLayoutDisplayPageProviderCheck.testjava");
 	}
 
+	@Test
+	public void testUpgradeJavaLayoutServicesCheck() throws Exception {
+		test("upgrade/UpgradeJavaLayoutServicesCheck.testjava");
+	}
+
```

Comment:
@TamyrisTorres, when adding a message we should test if the message is been sent. Take a look at the other tests.